pysat/instruments/pysat_testing2d.py

```diff
@@ -89,33 +96,27 @@ def load(fnames, tag=None, sat_id=None, malformed_index=False):
     # going to presume there are 5820 seconds per orbit (97 minute period)
     time_delta = date - pysat.datetime(2009, 1, 1)
     # mlt runs 0-24 each orbit.
-    data['mlt'] = test.generate_fake_data(time_delta.total_seconds(),
-                                          np.arange(num)*scalar,
-                                          period=5820, data_range=[0.0, 24.0])
+    data['mlt'] = mm_test.generate_fake_data(time_delta.total_seconds(), uts,
+                                             period=5820,
```

Comment:
The very specific way data is chosen should be stated and justified in the docstring.

Reply:
periods refer to 97 minutes and 104 minutes respectively.  Adding a note.